datasets/restaurant.py

```diff
@@ -196,8 +178,8 @@ def check_recalls_drink(self, drinks: list[str]):
         self.reasoning.append(f"The agent {recall_str} what it was drinking.")
         self.score += int(recalls)
 
-    def gpt_bool_check(self, context: LLMContext, key: str) -> bool:
-        eval_json = self.ask_llm(context)
+    def gpt_bool_check(self, context: LLMContext, key: str, **llm_kwargs) -> bool:
+        eval_json = self.ask_llm(context, GPT_CHEAPEST, **llm_kwargs)
```

Comment:
```
  File "/home/joe/Repos/goodai-ltm-benchmark/datasets/restaurant.py", line 161, in check_role_following
    follows_role = self.gpt_bool_check(context, "follows_role", model=GPT_4_TURBO_BEST)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/joe/Repos/goodai-ltm-benchmark/datasets/restaurant.py", line 182, in gpt_bool_check
    eval_json = self.ask_llm(context, GPT_CHEAPEST, **llm_kwargs)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: DynamicExample.ask_llm() got multiple values for argument 'model'
```

Reply:
The issue here is that the calling function is `follows_role = self.gpt_bool_check(context, "follows_role", model=GPT_4_TURBO_BEST)` and that wraps up GPT4 in  `**llm_kwargs`, but you are also passing `GPT_CHEAPEST` which is why the error is thrown